internal/testing/e2e/consul.go

```diff
@@ -531,12 +534,12 @@ func CreateConsulACLPolicy(ctx context.Context, cfg *envconf.Config) (context.Co
 		return ctx, nil
 	}
 	env := consulEnvironment.(*consulTestEnvironment)
-	token, _, err := env.consulClient.ACL().Bootstrap()
+	token, _, err := env.consulClient.Internal().ACL().Bootstrap()
```

Comment:
Could we add a `ACL()` method to the client wrapper since we plan to remove the `Internal()` catch-all soon?